Update set-up-database.rst

hanifmusa96 opened the pull request:
Update procedure to setup PostgreSQL database

<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of an existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->



<!-- Please keep an empty line above the dashes. -->
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

hanifmusa96 committed:
Update set-up-database.rst

Update procedure to setup PostgreSQL database

kaxil commented on docs/apache-airflow/howto/set-up-database.rst:
```diff
@@ -167,6 +167,7 @@ In the example below, a database ``airflow_db`` and user  with username ``airflo
    CREATE USER airflow_user WITH PASSWORD 'airflow_pass';
    GRANT ALL PRIVILEGES ON DATABASE airflow_db TO airflow_user;
    -- PostgreSQL 15 requires additional privileges:
+   \c airflow_db;
```
This isn't SQL statement

hanifmusa96 replied:
meta-command is acceptable in psql 
reference: https://www.postgresql.org/docs/current/app-psql.html

hanifmusa96 replied:
another extra reference:
- https://stackoverflow.com/questions/55590601/grant-privileges-to-a-specific-table-with-postgresql-relation-does-not-exis

kaxil replied:
psql is a cli tool to interact with Postgres where you can run and do more things than query. The code block here is pure SQL statement.

kaxil replied:
Meta-commands aren't SQL :)

kaxil replied:
If we merge this, users who copy paste their code to their sql tools / automation -- will have failures

hanifmusa96 committed:
Merge branch 'main' into docs/setup-postgresql-db

hanifmusa96 committed:
Merge branch 'main' into docs/setup-postgresql-db

hanifmusa96 committed:
Update set-up-database.rst

Provide a note to the user to connect to the airflow_db database before running the schema-specific grant statement.

hanifmusa96 committed:
Merge branch 'main' into docs/setup-postgresql-db
